extensions/SimonShiki/scopeVar.js

```diff
@@ -0,0 +1,152 @@
+// Name: Scope Variables
+// ID: scopeVar
+// Description: Manage your data inside your stack (or substack).
+// By: SimonShiki <https://scratch.mit.edu/users/SinanGentoo/>
+// Original: Skyhigh173
+
+(function (Scratch) {
+  "use strict";
+  if (!Scratch.extensions.unsandboxed) {
+    throw new Error(`
+            "Scope Variables" must be loaded in a unsandboxed environment.
+        `);
+  }
+  const { BlockType, ArgumentType, Cast } = Scratch;
+
+  class ScopeVar {
+    getInfo() {
+      return {
+        id: "scopeVar",
```

Comment:
Typically, extension IDs should contain some way to identify the creator, like your name or initials. "simonScopeVar" would be an example.